extensions/amp-analytics/0.1/vendors/snowplow_v2.json

```diff
@@ -36,6 +36,7 @@
   },
   "extraUrlParams": {
     "url": "${ampdocUrl}",
+    "tna": "${nameTracker}",
```

Comment:
I appreciate it's called `name_tracker` in the warehouse but it's a bit of an odd name. Does `trackerName` feel more natural or `namespace`?

Reply:
Namespace I _think_ is used for the scope of a config within the AMP domain, so not sure on that one.

I'm totally ok to change to `trackerName`, I went with `nameTracker` just to keep it consistent with how it's named in the tracker protocol/docs.

Reply:
Let's leave it as nameTracker then, it doesn't make much difference either way

Reply:
Oh, cool will do. :)